transmit-lib/src/cmr/transmit/urs.clj

```diff
@@ -1,82 +1,63 @@
 (ns cmr.transmit.urs
   (:require
-   [clojure.data.xml :as x]
-   [cmr.common.mime-types :as mt]
    [cmr.common.services.errors :as errors]
-   [cmr.common.xml :as cx]
    [cmr.transmit.config :as config]
    [cmr.transmit.connection :as conn]
-   [cmr.transmit.http-helper :as h]
+   [cmr.transmit.http-helper :as http-helper]
    [ring.util.codec :as codec]))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; URL functions
 
 (defn- get-user-url
   [conn username]
-  (format "%s/users/%s" (conn/root-url conn) username))
+  (format "%s/api/users/verify_uid?uid=%s" (conn/root-url conn) username))
 
-(defn login-url
+(defn- login-application-url
   [conn]
-  (format "%s/login" (conn/root-url conn)))
+  (format "%s/oauth/token?grant_type=client_credentials" (conn/root-url conn)))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Request functions
 
+(defn- get-bearer-token
```

Comment:
Were you able to test login and get user URL successfully against SIT URS?

Reply:
Locally yes, I need to change some SIT configs when this goes there though.